sdk/eventhub/eventhubs-checkpointstore-blob/samples/receiveEventsUsingCheckpointStore.ts

```diff
@@ -0,0 +1,77 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how to use the EventHubConsumerClient to process events from all partitions
+  of a consumer group in an Event Hubs instance. It also demonstrates the process of checkpointing an event
+  which helps new instances of your application that may have spun up for scaling or for crash recovery.
```

Comment:
helps how?

Reply:
Updated the description a bit. Let me know if it reads better.